draft-ietf-quic-transport.md

```diff
@@ -1475,6 +1475,48 @@ lifetime of a connection, especially in response to connection migration
 ({{migration}}); see {{issue-cid}} for details.
 
 
+## Authenticating Connection IDs {#cid-auth}
+
+The choice each endpoint makes about connection IDs during the handshake is
+authenticated by including all values in transport parameters; see
+{{transport-parameters}}. This ensures that all connection IDs are authenticated
+by the cryptographic handshake.
+
+Each endpoint includes the value of the Source Connection ID field from Initial
+packets it sends in the handshake_connection_id transport parameter; see
```

Comment:
One could read that this Connection ID is only used on Initial packets.  If so, I think initial_connection_id would be a better TP name.  But I think this CID can be used used on Initial and 0-RTT from the client and all Initial, Handshake and 1-RTT from the server?

Reply:
I think it's called `handshake_connection_id` because it is the connection ID used during the handshake, I think it's a reasonable name

Reply:
In keeping with retry_connection_id being the SCID that is used on Retry packets, I have a preference for initial_connection_id, since it is the SCID used on Initial packets. 

In the same vein, and this is a weaker preference, I would propose changing original_connection_id to original_initial_connection_id. I think this makes it clear that this might have been initial_connection_id had it not been for a Retry.